packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/objects/ObjectDefinitionBuilder.kt

```diff
@@ -253,6 +253,12 @@ open class ObjectDefinitionBuilder {
    */
   fun Events(vararg events: String) {
     eventsDefinition = EventsDefinition(events)
+
+    // Register stub functions to bypass react-native `NativeEventEmitter` warnings
+    // WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method.
+    // WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.
+    Function("addListener") { }
+    Function("removeListeners") { }
```

Comment:
We could in theory check if those functions weren't added by the user, but I'm not sure if that is needed. What do you think?

Reply:
nice suggestion! https://github.com/expo/expo/pull/19920/commits/7ab54f02e165b0d91c33ff555f2583237437fc2d updates this. does the code change make sense to you?